inlang/source-code/sdk/src/createNodeishFsWithAbsolutePaths.ts

```diff
@@ -19,14 +19,14 @@ export const createNodeishFsWithAbsolutePaths = (args: {
 
 	// get the base path of the settings file by
 	// removing the file name from the path
-	const bathPath = args.settingsFilePath.split("/").slice(0, -1).join("/")
+	const basePath = args.settingsFilePath.split(/[/\\]/).slice(0, -1).join("/")
```

Comment:
why are you doing this? The path should be normalized at this point. And a windows path doesn't even seem to use `\\`

Reply:
it's the same expression like here https://github.com/inlang/monorepo/blob/029d78dc877dbcb3273dd617a709a06df467dc9d/inlang/source-code/sdk/src/createNodeishFsWithAbsolutePaths.ts & no, it's not normalized at this point.

Reply:
fixed with normalizing path before.